### DISCUSSION TITLE
Little Request for chat rooms - /chat see

### DISCUSSION THREAD

**Discussion by myavuzokumus** [Zrips/CMI#6043] [Enhancement]
Can add permission for see chat rooms message when not joined?

Or a similar command?

Because while I see the messages in the rooms, I want to have a global chat at the same time.

Follow a new aliases to /chat command, example:

/chat see <room-name>

**Comment by Zrips** [comment]
Will add, tho we do need 2 commands, one to start watching and one to stop and maybe even one to list all rooms you are in as a watcher with. So yea, 3 commands in total then.
